src/makeRobotAccountant.js

```diff
@@ -6,7 +6,22 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  const errorMessage = 'Bzzz... Error!';
+
+  return (num1) => {
+    counter++;
+
+    const isMoreThen3 = counter > 3;
+    const isOdd = counter % 2 === 0;
+
+    const messageOrNum = isMoreThen3 && isOdd
+      ? message => errorMessage
+      : num2 => num1 + num2;
+
+    return messageOrNum;
```

Comment:
We don't need to create an extra variable if we just return the functions we want.
```suggestion
    if (isMoreThen3 && isOdd) {
      return () => errorMessage;
    }
    
    return num2 => num1 + num2;
```